tap_github/client.py

```diff
@@ -84,13 +84,13 @@ def get_next_page_token(
         # Unfortunately endpoints such as /starred, /stargazers, /events and /pulls do not support
         # the "since" parameter out of the box. So we use a workaround here to exit early.
         # For such streams, we sort by descending dates (most recent first), and paginate
-        # "back in time" until we reach records before our "since" parameter.
+        # "back in time" until we reach records before our "fake_since" parameter.
         request_parameters = parse_qs(str(urlparse(response.request.url).query))
         # parse_qs interprets "+" as a space, revert this to keep an aware datetime
         try:
             since = (
-                request_parameters["since"][0].replace(" ", "+")
-                if "since" in request_parameters
+                request_parameters["fake_since"][0].replace(" ", "+")
```

Comment:
Is this not breaking other streams that actually support the `since` parameter?

Reply:
the streams that support the since parameter don't need to exit early, because the since parameter takes care of the filtering server-side already :)

Reply:
Maybe worth a short comment in the code?